docs/user/registering-microshift-devices-acm.md

```diff
@@ -92,29 +95,36 @@ As described in the user documentation, a fleet specification is composed of var
 1. **Configuration**: The configuration section uses the HTTP configuration provider to fetch information from an endpoint. The repository `acm-test` contains the registration URL for ACM's agent registration:
 
     ```console
-    - configType: HttpConfigProviderSpec
+    - name: acm-crdv
```

Comment:
```suggestion
    - name: acm-crd
```

Reply:
Fixed typo.
Yes, it's expected that it's owned by root.